src/SFA.DAS.EmployerIncentives.Domain.UnitTests/ApprenticeshipIncentive/WhenRequestingEmploymentChecks.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Linq;
+using AutoFixture;
+using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Events;
+using SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.Models;
+using SFA.DAS.EmployerIncentives.Domain.ValueObjects;
+using SFA.DAS.EmployerIncentives.Enums;
+
+namespace SFA.DAS.EmployerIncentives.Domain.UnitTests.ApprenticeshipIncentive
```

Comment:
If we change the namespace to `SFA.DAS.EmployerIncentives.Domain.UnitTests.ApprenticeshipIncentiveTests` in line with the other unit tests, then there's no need for all the instances of the namespace prefix in code, i.e.
`private ApprenticeshipIncentives.ApprenticeshipIncentive`